test/jit/test_slice.py

```diff
@@ -136,3 +136,25 @@ def tuple_slice(a):
         self.assertTrue(num_outputs == {2})
         self.run_pass('lower_all_tuples', tuple_graph)
         self.assertTrue('Tuple' not in str(tuple_graph))
+
+    def test_module_list_slicing(self):
+        class Foo(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                m1 = torch.nn.Linear(1, 2)
+                m2 = torch.nn.ReLU()
+                m3 = torch.nn.GRU(1, 2)
+                self.test = torch.nn.ModuleList([m1, m2, m3])
+
+            def forward(self):
+                return self.test[::-2], self.test[1:2:]
```

Comment:
These are not great test data.

For example, `[1:2:]` is bad for testing step size, any `step` size would result in same slicing behavior.

```
>>> a = [1, 2, 3]
>>> a[1:2:3]
[2]
>>> a[1:2:2]
[2]
>>> a[1:2:1]
[2]
```

Reply:
done